### DISCUSSION TITLE
Excessive whitespace in comments

### DISCUSSION THREAD

**Discussion by slondr** [community/community#40040] [Mobile]
Every now and then I see a giant amount of whitespace within a comment, after the end of the text…about 8x to 10x my device height. This does not happen very often or consistently, and is solved by a refresh, but I've seen it about a dozen times in the last few months. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.90.0 (84076705)
Device: iPhone Xʀ (iOS 16.1.1)
</details>

**Comment by ossanna16** [comment]
Hi there @slondr and welcome to our community! Thank you for reporting this. Could you perhaps add a screenshot? Thank you!

**Reply by slondr** [reply]
I tried to capture the phenomena the best I could, which requires a few screenshots:

![image](https://user-images.githubusercontent.com/13040688/204414047-d1155e0e-1a37-4ab7-a42e-b4565fd9b63d.png)
![image](https://user-images.githubusercontent.com/13040688/204414058-2d234731-ae11-4ba3-8dff-5713af830861.png)
![image](https://user-images.githubusercontent.com/13040688/204414063-3f558592-d2e3-43fc-93a6-8a76ba60ef52.png)
![image](https://user-images.githubusercontent.com/13040688/204414077-d5d5378c-1191-4694-8b19-1673c4d4c789.png)

**Reply by amadejpapez** [reply]
Also reported here https://github.com/orgs/community/discussions/38207

I am seeing this issue almost every day now

**Reply by amadejpapez** [reply]
Looks like this is fixed in todays update!
